nodejs/image.bzl

```diff
@@ -18,7 +18,9 @@ The signature of this rule is compatible with nodejs_binary.
 
 load(
     "//lang:image.bzl",
-    "app_layer",
+    "app_layer_attrs",
+    "app_layer_impl",
```

Comment:
can you alias these with lang_ so that they are visibly distinct from the ones declared here